src/test/java/ca/on/oicr/gsi/dimsum/service/filtering/SampleSortTest.java

```diff
@@ -28,29 +31,81 @@ public void testSortByNameDescending() {
     assertOrder(samples, Sample::getName, namesOrdered, true);
   }
 
+  @Test
+  public void testSortByQcStatusAscending() {
+    List<Sample> samples = getSamplesSorted(SampleSort.QC_STATUS, false);
+    assertOrder(samples, SampleSort::getSampleQcStatus, qcStatusOrdered, false);
```

Comment:
This is testing that the statuses are ordered, but not testing that the correct status ints are being returned for each sample. Maybe leave this as is, but also add tests for `SampleSort#getSampleQcStatus`. They should look something like:

* mock sample
* mock all relevant attributes, similar to what you did in `mockSample` below
* assert that `getSampleSort` returns the expected value for that sample

Include one for every possible status int.